### PR TITLE
Add some better sample env vars in settings.json

### PR DESCRIPTION
**Pull request by aeisenberg**
Small changes that will help with debugging custom codeql instances.

## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -44,10 +44,18 @@
   },
   "jestrunner.debugOptions": {
     // Uncomment to debug integration tests
-    // "attachSimplePort": 9223,
+    "attachSimplePort": 9223,
```
Comment (aeisenberg) [inline .vscode/settings.json:47]: This should remain commented.
```suggestion
    // "attachSimplePort": 9223,
```

**Review by koesie10 — APPROVED** [review]
LGTM

File: .vscode/settings.json [hunk 2]
```diff
@@ -48,6 +48,14 @@
     "env": {
       "LANG": "en-US",
       "TZ": "UTC",
+
+      // Uncomment to set a custom path to a CodeQL checkout.
+      // "TEST_CODEQL_PATH": "../codeql",
```
Comment (elenatanasoiu) [inline .vscode/settings.json:53]: Do we still need this in [launch.json](https://github.com/github/vscode-codeql/blob/main/.vscode/launch.json#L147)?

Comment (koesie10) [reply .vscode/settings.json:53]: Yes, the `launch.json` is used when you run the "Launch Integration Tests - With CLI" action using the "Run & Debug" panel of VSCode. These environment variables are instead used when you use the Jest runner to run tests. These are completely separate.

**Pull request merged**